features/jira.py

```diff
@@ -0,0 +1,27 @@
+from datetime import date
+from jira import JIRA
+from pprint import pprint, pformat
+import settings
+
+if settings.CREATE_JIRA_VERSIONS == "True":
+    jira_options = {
+        'server': settings.JIRA_SERVER
+    }
+    jira = JIRA(options=jira_options, basic_auth=(settings.JIRA_USERNAME, settings.JIRA_PASSWORD))
```

Comment:
Same risk of namespace clash here, although you've mitigated by the way you've done the import (i.e. you've imported the `JIRA` object, and not the core `jira` package)